dvc/stage/serialize.py

```diff
@@ -59,15 +61,18 @@ def _serialize_out(out):
 
 @no_type_check
 def _serialize_outs(outputs: List[BaseOutput]):
-    outs, metrics, plots = [], [], []
+    outs, metrics, plots, live = [], [], [], None
     for out in sort_by_path(outputs):
         bucket = outs
         if out.plot:
             bucket = plots
         elif out.metric:
             bucket = metrics
+        elif out.live:
+            live = _serialize_out(out)
```

Comment:
Just as an extra caution
```suggestion
            assert live is None
            live = _serialize_out(out)
```